frontend/src/lib/components/LemonCalendar/LemonCalendar.scss

```diff
@@ -0,0 +1,15 @@
+.LemonCalendar {
+    .LemonCalendar__today {
+        position: relative;
+        &:after {
```

Comment:
Nit: pseudo-elements are preceded by a double colon (`::after`, `::before`), though I guess browser don't care that much because backwards compatibility